lib/user_auth/screens/auth_home_screen.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
All screens should be in one folder hierarchy i.e "lib/screens"

All login and sign-up screens probably to be moved to "lib/screens/account/login"